src/index.ts

```diff
@@ -164,6 +170,29 @@ function sendTransactions(txs: Transaction[], requestId?: RequestId): SentSDKMes
   return message;
 }
 
+/**
+ * Request Safe app to send transactions
+ * @param txs
+ */
+function sendTransactionsWithParams(
+  txs: Transaction[],
+  params?: SendTransactionParams,
+  requestId?: RequestId,
+): SentSDKMessage<'SEND_TRANSACTIONS_V2'> {
+  if (!txs || !txs.length) {
+    throw new Error('sendTransactions: No transactions were passed');
```

Comment:
Should we refactor the function name in this error too?

Reply:
Yep :)